### PR TITLE
Refactor: models finalize screen

### PR DESCRIPTION
**Pull request by macintushar**
## Description

- Changed Finalize Model screen to have same width and color as other screens

image

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would impact existing functionality)
- [ ] Styling change
- [ ] Documentation update
- [x] Refactoring
- [ ] Chore
  
## How Has This Been Tested?

None

## Checklist:
- [x] Ensure a branch name is prefixed with `feature`, `bugfix`, `hotfix`, `release`, `style` or `chore` followed by `/` and branch name e.g `feature/add-salesforce-connector`
- [ ] Added unit tests for the changes made (if required)
- [x] Have you made sure the commit messages meets the guidelines?
- [x] Added relevant screenshots for the changes
- [x] Have you tested the changes on local/staging?
- [x] Have you made sure the code you have written follows the best practises to the best of your knowledge?

### CODE REVIEW

File: .github/workflows/ui-ci.yml [hunk 1]
```diff
@@ -33,11 +33,11 @@ jobs:
...
         run: |
-          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./ui/cc-test-reporter
-          chmod +x ./ui/cc-test-reporter
-          ./ui/cc-test-reporter before-build
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+          chmod +x ./cc-test-reporter
+          ./cc-test-reporter before-build
           if [ -f coverage/lcov.info ]; then
-            ./ui/cc-test-reporter after-build --exit-code $? --coverage-input-type lcov
+            ./cc-test-reporter after-build --exit-code $? --coverage-input-type lcov
```
Comment (sumitd94) [inline .github/workflows/ui-ci.yml:40]: I think we do need /ui

**Comment by karthik-sivadas** [comment]
@macintushar This is still not correct based on the image shared above. There is an alignment issue in the above image
Screenshot 2024-04-20 at 1 22 19 PM

File: .github/workflows/ui-ci.yml (hunk 1 above)
Comment (karthik-sivadas) [reply .github/workflows/ui-ci.yml:40]: @macintushar Both works as we just need to run the cc from where it's downloaded

Comment (macintushar) [reply .github/workflows/ui-ci.yml:40]: @karthik-sivadas should it be changed or is it okay?

Comment (karthik-sivadas) [reply .github/workflows/ui-ci.yml:40]: It is ok to leave it as it is as we are getting the code climate

**Review by karthik-sivadas — APPROVED** [review]

File: .github/workflows/ui-ci.yml (hunk 1 above)
Comment (macintushar) [reply .github/workflows/ui-ci.yml:40]: Understood, merging

**Pull request merged**